server.js

```diff
@@ -0,0 +1,11 @@
+const express=require("express");
+const app=express();
+const port=3000

```

Comment:
Consider using environment variables for the port number. This allows for better configurability and security, especially when deploying the application. You can use packages like 'dotenv' to manage environment variables. [important]